inc/core/filters/media-text-enhanced.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Bcgov\Theme\Block;
+
+/**
+ * Improve button block by adding additional attribtes: size class and tabindex.
+ *
+ * Load this file via inc/core/theme-bootstrap.php.
+ *
+ * @param  string $block_content Block content to be rendered.
+ * @param  array  $block         Block attributes.
+ * @return string
+ */
+function add_media_text_attributes( $block_content = '', $block = [] ) {
+
+	if ( isset( $block['blockName'] ) && 'core/media-text' === $block['blockName'] ) {
+		$defaults = [
+			'title' => 'none',
+		];
+
+		$args = wp_parse_args( $block['attrs'], $defaults );
+
+		if ( 'none' !== $args['title'] ) {
+			$title = 'title="' . esc_attr( $args['title'] ) . '"';
+		} else {
+			$title = null;
```

Comment:
just making sure this is your desired results as title = null. the logic afterwards seems a bit weird to at <img null

Reply:
It is, but it could be structured differently for sure.